website/activemembers/gsuite.py

```diff
@@ -22,13 +24,17 @@ def directory_api(self):
             return self._directory_api
         return get_directory_api()
 
+    def _generate_password(self, member: Member):
+        alphabet = string.ascii_letters + string.digits + string.punctuation
```

Comment:
Let's not use all of the punctuation options. Some of them are pretty weird for in a password.

```suggestion
        alphabet = string.ascii_letters + string.digits + "!@#$%^&*-_=+?"
```